package.json

```diff
@@ -123,6 +124,7 @@
     },
     "dependencies": {
         "dayjs": "^1.8.10",
+        "pikaday": "^1.8.2",
```

Comment:
I know this is copied from Twist, but reactist should set higher standards. [pikaday is no longer being maintained](https://github.com/Pikaday/Pikaday/issues/884) with over 200 open issues – I don't think we should continue using it and rather research alternatives.

Reply:
Ugh, ok. Were there any investigations into this already done for Twist?

Reply:
Not that I'm aware of. In Todoist we use a custom implementation because the calendar needs to be a bit more involved. Not sure what's best to use maybe @gnapse knows a good implementation of a simple date picker?
Maybe we could use the native calendar? – looks a bit shabby but could work for these one-off selection scenarios 🤷‍♂️

Reply:
> Maybe we could use the native calendar?

@henningmu The problem with the native date picker is that it can't be styled. So if you are in the dark theme in twist, the calendar always appears with a white background. It's one of the main reasons I opted to try and do the date picker like this (I was anticipating design asking for it to be styled).

Reply:
@gnapse Any thoughts on this? Any recommendations for datepickers you have?

Reply:
Not off the top of my head. Can I suggest, before we make a rush decision here, to leave out this from Reactist, and for the time being, add it to the context where you need it in the short term?

Reply:
Ok, will remove it for now.